runtime/src/accounts_db.rs

```diff
@@ -1047,46 +1062,118 @@ impl AccountsDB {
         datapoint_info!("accounts_db-stores", ("total_count", total_count, i64));
     }
 
-    pub fn verify_bank_hash(
+    fn accumulate_account_hashes(mut hashes: Vec<(Pubkey, Hash)>) -> Hash {
+        let mut sort = Measure::start("sort");
+        hashes.par_sort_by(|a, b| a.0.cmp(&b.0));
+        sort.stop();
+        let mut hash_time = Measure::start("hash");
+        let hashes: Vec<_> = hashes.chunks(16).collect();
```

Comment:
Just curious: Is there any special reasoning for choosing `16`? Bench result? I mean, why not 8, 32 or 64?

Reply:
I did some tests, on my machine there is a speedup until about 128-256, so I will go with 256 fanout.